### DISCUSSION TITLE
Alguém sabe como pega a conquista "Quickdraw"?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#20683] [Other Feature Feedback, Questions, & Ideas] [answered]
Eu estou tentando pegar a conquista de Quickdraw, mas não sei

**Comment by Kayke-Fujinaka** [comment, marked as answer]
Você pode criar uma branch e então mudar para ela. Logo após isso, você certifica que está realmente na branch e utiliza os comandos normais para subir o projeto para o Github. 

O projeto que você realizou isso vai estar com um pull request notificado. Aí, você vai entra nele e depois cancela. Porém tem que ser em 5 minutos isso.
